conference/src/main/java/com/kyriba/conference/management/domain/Hall.java

```diff
@@ -1,18 +1,24 @@
 package com.kyriba.conference.management.domain;
 
 
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.ToString;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
 
-//@Entity
-@Getter
-@ToString
-@EqualsAndHashCode
+
+@Entity
+@Data
```

Comment:
Data brings equals/hashCode that are dangerous for entity

Reply:
fixed to @Getter and @Setter